src/handlers/internal-server-error-handler.ts

```diff
@@ -29,12 +29,16 @@ const serverErrorHandler: ErrorRequestHandler = (err, req, res, next) => {
     return next(err);
   }
 
+  if (res.statusCode === 503) {
+      return res.render(getHtmlPath("errors", "service-unavailable-s3", ERROR_PAGES.SERVICE_UNAVAILABLE))
```

Comment:
Do you know if this will cache the template or load it from disk each time? That's something we probably want to watch out for when implementing for real

Reply:
Which is interesting because it implies that lower envs won't cache the views. Which might not help the results of the perf tests.

Reply:
http://expressjs.com/en/5x/api.html#res.render

Reply:
Ah you're right - i just looked and it is production across the envs.

And yeah render works fine - but I guess there might be some perf gains for using send... I'll have a quick look

Reply:
So it looks like sendFile _won't_ cache the file in memory - it'll read it from disk each time. So render might actually be a better choice, unless you want to introduce some other sort of caching.

Although I notice that we're globally setting `Cache-control private no-store` (among others) on all responses. Maybe if we allow caching, cloudfront will handle it for us?

Reply:
Ah - but we probably don't want CloudFront caching `/page/ipv-success-page` as a 503 error screen

If render 'just works' then that might be easiest. Otherwise we can load the file on startup and serve it from memory.